packages/mdx/package.json

```diff
@@ -35,14 +36,16 @@
     "mdxast"
   ],
   "scripts": {
-    "test-api": "jest test --coverage",
+    "test-api": "jest test",
+    "test-coverage": "jest test --coverage",
     "test-types": "dtslint types",
-    "test": "npm run test-api && npm run test-types"
+    "test": "yarn test-coverage && yarn test-types"
   },
   "dependencies": {
     "@babel/core": "7.10.5",
     "@babel/plugin-syntax-jsx": "7.10.4",
     "@babel/plugin-syntax-object-rest-spread": "7.8.3",
+    "@mdx-js/util": "2.0.0-next.1",
```

Comment:
Is this pinned to next 1 for a reason? `2.0.0-next.8` is the latest published in this release line.

Reply:
Somewhat complex to explain the background:
I’m not 100% sure which, but some gatsby-mdx package uses `@mdx-js/mdx/util`, which used to be where the utilities were, which were later moved to `@mdx-js/util`. I’ve deprecated that package. But as we’re using a linked package, gatsby was failing because it was missing.

I added a warning to `@mdx-js/mdx/util` and propose actually removing it (related to https://github.com/mdx-js/mdx/pull/1338), but can’t for now.
The reason for using a lower version was to ensure that lerna doesn’t update it on publish, maybe there’s another way around it.

Reply:
:raised_eyebrow: won't lerna see an old version and update it anyway?

Reply:
The v1 versions of a couple packages were used in the main `package.json`, and didn’t get updated apparently. Might have to do with only majors, I dunno 🤷‍♂️